IconDiffBot/Core/Application.cs

```diff
@@ -57,12 +59,15 @@ public void ConfigureServices(IServiceCollection services)
 					builder.UseSqlServerStorage(dbConfig.ConnectionString, new SqlServerStorageOptions { PrepareSchemaIfNecessary = true });
 			});
 
+			services.AddHangfireServer();
+
 			services.Configure<IISOptions>((options) => options.ForwardClientCertificate = false);
-			services.AddMvc();
 			services.AddOptions();
 			services.AddLocalization();
+			services.AddControllersWithViews();
+			services.AddRazorPages();
```

Comment:
Why'd you change AddMvc with these two?

Reply:
I needed to change it to these two for it to work, and the migration said guide said so